src/Symfony/Component/Yaml/Inline.php

```diff
@@ -107,6 +109,10 @@ public static function parse($value, $flags = 0, $references = array())
                 $result = self::parseScalar($value, $flags, null, $i, true, $references);
```

Comment:
Is it safe to assume that we can always evaluate the scalar even when we are parsing the value of a tag?